PWGLF/Tasks/lambdakzeroanalysisMC.cxx

```diff
@@ -241,23 +260,20 @@ struct lambdakzeroParticleCountMC {
             registry.fill(HIST("hK0ShortCount_PtDiff"), mcparticle.pt());
             if ((mcparticle.daughter0_as<aod::McParticles>().pdgCode() == 211 && mcparticle.daughter1_as<aod::McParticles>().pdgCode() == -211) || (mcparticle.daughter0_as<aod::McParticles>().pdgCode() == -211 && mcparticle.daughter1_as<aod::McParticles>().pdgCode() == 211)) {
               registry.fill(HIST("hK0ShortCount"), 1.5);
-              registry.fill(HIST("hK0ShortCount_PtDiff"), mcparticle.pt());
```

Comment:
Sorry if I miss something, but did we move the (very necesssary) pT-differential histograms somewhere else?

Reply:
Ah! Good spotting, I had the histogram filled twiced (before and after the daughter mc PID check), wanted to removed the double counting, although I removed the one I wanted to keep. Will fix this immediately